Godnoken/Newsfeed-Components
# Components 1 - LambdaSchool

## What does this PR do?
Newsfeed Components I Project MVP
​
## Description of tasks completed
- Implemented articles component and added it to the DOM
- Implemented menu component and added it to the DOM
​
## How can this be tested?
Clone the project.
Open up the project files in your favourite code editor.
Run index.html to see components article and menu implemented.
Check the menu.js and article.js for component code.
​
## Relevant screenshot
![image](https://user-images.githubusercontent.com/17499594/67865562-c2158f80-fb27-11e9-97b6-76513d6dd1f9.png)

![image](https://user-images.githubusercontent.com/17499594/67865615-d9ed1380-fb27-11e9-87e4-224f9ada8ed2.png)
